hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/SnapshotChainManager.java

```diff
@@ -246,7 +257,7 @@ private boolean deleteSnapshotPath(String snapshotPath,
     return status;
   }
 
-  private void loadFromSnapshotInfoTable(OMMetadataManager metadataManager)
+  public void loadFromSnapshotInfoTable(OMMetadataManager metadataManager)
```

Comment:
Do we need to clear the new map in this method as well?

```
snapshotPathToTableKey.clear();
```

and what about `latestPathSnapshotID`

Reply:
I think it should be okay to clear it. As it is reconstructed again.